Support SunOS ar and SunOS-specific GCC behaviour.

--- mesonbuild/environment.py	2018-09-22 15:22:03.000000000 +0000
+++ mesonbuild/environment.py	2018-10-13 17:01:02.934603234 +0000
@@ -445,6 +445,8 @@ class Environment:
         # Detect GCC type (Apple, MinGW, Cygwin, Unix)
         if '__APPLE__' in defines:
             return CompilerType.GCC_OSX
+        elif '__sun' in defines:
+            return CompilerType.GCC_SUNOS
         elif '__MINGW32__' in defines or '__MINGW64__' in defines:
             return CompilerType.GCC_MINGW
         elif '__CYGWIN__' in defines:
@@ -914,6 +916,8 @@ This is probably wrong, it should always
                 return ArLinker(linker)
             if p.returncode == 1 and err.startswith('usage'): # OSX
                 return ArLinker(linker)
+            if p.returncode == 1 and err.startswith('ar: bad option'): # SunOS
+                return ArLinker(linker)
         self._handle_exceptions(popen_exceptions, linkers, 'linker')
         raise EnvironmentException('Unknown static linker "%s"' % ' '.join(linkers))
 
