--- firefox-66.0.3/browser/branding/official/pref/firefox-branding.js	2019-04-09 21:59:43.000000000 +0000
+++ firefox-66.0.3/browser/branding/official/pref/firefox-branding.js.new	2019-04-28 18:43:20.126250474 +0000
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 pref("startup.homepage_override_url", "");
-pref("startup.homepage_welcome_url", "about:welcome");
+pref("startup.homepage_welcome_url", "file:///usr/share/doc/openindiana-welcome/html/index.html");
 pref("startup.homepage_welcome_url.additional", "");
 // Interval: Time between checks for a new version (in seconds)
 pref("app.update.interval", 43200); // 12 hours
