--- firefox-66.0.3/js/xpconnect/src/XPCWrappedNative.cpp	2019-04-09 22:00:01.000000000 +0000
+++ firefox-66.0.3/js/xpconnect/src/XPCWrappedNative.cpp.new	2019-04-28 22:11:35.059546303 +0000
@@ -6,14 +6,6 @@
 
 /* Wrapper object for reflecting native xpcom objects into JavaScript. */
 
-#if (__GNUC__ && __linux__ && __PPC64__ && _LITTLE_ENDIAN)
-// Stack protection generates incorrect code currently with gcc on ppc64le
-// (bug 1512162).
-#  define MOZ_GCC_STACK_PROTECTOR_DISABLED 1  // removed at end of file
-#  pragma GCC push_options
-#  pragma GCC optimize("no-stack-protector")
-#endif
-
 #include "xpcprivate.h"
 #include "mozilla/jsipc/CrossProcessObjectWrappers.h"
 #include "nsWrapperCacheInlines.h"
