--- rustc-1.28.0-src/src/librustc_codegen_llvm/back/linker.rs	2018-07-31 00:15:53.000000000 +0000
+++ rustc-1.28.0-src/src/librustc_codegen_llvm/back/linker.rs	2018-08-22 08:14:19.167640346 +0000
@@ -320,7 +320,7 @@ impl<'a> Linker for GccLinker<'a> {
 
     fn no_default_libraries(&mut self) {
         if !self.is_ld {
-            self.cmd.arg("-nodefaultlibs");
+            self.cmd.arg("-Wl,-znodefaultlibs");
         }
     }
