--- rustc-1.28.0-src/src/bootstrap/bin/rustc.rs	2018-07-31 00:15:53.000000000 +0000
+++ rustc-1.28.0-src/src/bootstrap/bin/rustc.rs.new	2018-08-22 08:18:48.349792845 +0000
@@ -233,7 +233,7 @@ fn main() {
                 cmd.arg("-Z").arg("osx-rpath-install-name");
                 Some("-Wl,-rpath,@loader_path/../lib")
             } else if !target.contains("windows") && !target.contains("wasm32") {
-                Some("-Wl,-rpath,$ORIGIN/../lib")
+                Some("-Wl,-rpath,/usr/lib/64")
             } else {
                 None
             };
